cobigen/cobigen-textmerger/src/test/java/com/devonfw/cobigen/textmerger/TextAppenderTest.java

```diff
@@ -9,310 +9,348 @@
 import org.junit.Test;
 
 import com.devonfw.cobigen.api.exception.MergeException;
-import com.devonfw.cobigen.textmerger.TextAppender;
 
 /**
  *
  * @author mbrunnli (03.06.2014)
  */
 public class TextAppenderTest {
 
-    /**
-     * Root path to all resources used in this test case
+  /**
+   * Root path to all resources used in this test case
+   */
+  private static String testFileRootPath = "src/test/resources/";
+  
+   /**
+     * Newline charecter for replacement.
      */
-    private static String testFileRootPath = "src/test/resources/";
+  private final static String ALL_LINE_ENDINGS = "\\r\\n|\\r|\\n";
 
-    /**
-     * Tests a merge without adding a new line before appending the patch
-     * @throws Exception
-     *             if errors occur while merging
-     * @author mbrunnli (03.06.2014)
-     */
-    @Test
-    public void testMerge_appendWithoutNewLineNoAnchors() throws Exception {
-        TextAppender appender = new TextAppender("textmerge_append", false);
-        String mergedString = appender.merge(new File(testFileRootPath + "BaseFile.txt"), "Test3", "UTF-8");
-        assertThat(mergedString).isEqualTo(FileUtils.readFileToString(new File(testFileRootPath + "MergedFile.txt")));
-    }
+  /**
+   * Tests a merge without adding a new line before appending the patch
+   *
+   * @throws Exception if errors occur while merging
+   * @author mbrunnli (03.06.2014)
+   */
+  @Test
+  public void testMerge_appendWithoutNewLineNoAnchors() throws Exception {
 
-    /**
-     * Tests a merge with adding a new line before appending the patch
-     * @throws Exception
-     *             if errors occur while merging
-     * @author mbrunnli (03.06.2014)
-     */
-    @Test
-    public void testMerge_appendWithNewLineNoAnchors() throws Exception {
-        TextAppender appender = new TextAppender("textmerge_append", true);
-        String mergedString = appender.merge(new File(testFileRootPath + "BaseFile.txt"), "Test3", "UTF-8");
-        assertThat(mergedString)
-            .isEqualTo(FileUtils.readFileToString(new File(testFileRootPath + "MergedFile_withNewLine.txt")));
-    }
+    TextAppender appender = new TextAppender("textmerge_append", false);
+    String mergedString = appender.merge(new File(testFileRootPath + "BaseFile.txt"), "Test3", "UTF-8");
+    mergedString = mergedString.replaceAll(ALL_LINE_ENDINGS, "");
+    assertThat(mergedString)
+        .isEqualTo(FileUtils.readFileToString(new File(testFileRootPath + "MergedFile.txt")).replaceAll(ALL_LINE_ENDINGS, ""));
+  }
```

Comment:
It seems that your last commit has broken the format. Can you check that?

Reply:
Hi Jaime, it also seems that on one branch multiple plugins were edited. Could you please check that? Is the setup of the ide right here? On one dev branch just sources of one plugin should be edited

Reply:
@sujithmn doesn't still have the IDE correctly configured. If I have time this week I will help him. Meanwhile please Sujith create one Pull Request per each plug-in: One for Java-plugin, one for property-plugin and one for text-merger. You will see that we have one branch per plug-in, from there we need to create our new branches.